fix(native): Correctly patch the dark class & parse colours more robu…

Covkie opened the pull request:
…stly (https://github.com/bunny-mod/BunnyXposed/pull/6)

* fix(native) Patch the right semantic class on dark

I guess no one noticed this lmao :p

I realized this while implementing unbound's native side themeing :33

* fix: Properly handle alpha when parsing colours

Ushie reviewed (changes requested):
This doesn't warrant a version bump, revert back to v1.2.0

poob1 commented:
There you go, reverted the Version Bump.

riivx reviewed (approved):
LGTM

Ushie commented:
Is the package name update necessary? it might be best to draft this until we get our domain, so we can update it properly at once following https://en.m.wikipedia.org/wiki/Reverse_domain_name_notation

poob1 commented:
Sure, that doesn't hurt. I'll make it a draft then.

Covkie committed:
fix(native): Correctly patch the dark class & parse colours more robustly (#6)

* fix(native) Patch the right semantic class on dark

I guess no one noticed this lmao :p

I realized this while implementing unbound's native side themeing :33

* fix: Properly handle alpha when parsing colours
